Exiled.Events/Patches/Generic/Scp173BeingLooked.cs

```diff
@@ -11,55 +11,144 @@ namespace Exiled.Events.Patches.Generic
     using System.Collections.Generic;
```

Comment:
All of the changes in this file need to be removed from this PR and put in their own, as the changes are not dependant on this PR and do not have anything to do with the other features implemented in this PR.

Reply:
Done https://github.com/Exiled-Team/EXILED/pull/1239